tests/test_vision_data.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from fastai import *
+from fastai.vision import *
+
+@pytest.fixture(scope="module")
+def path(request):
+    path = untar_data(URLs.MNIST_TINY)
+    d = defaults.device
+    defaults.device = torch.device('cpu')
```

Comment:
Why impose a device? The tests are run on cpu only and GPU machines so you don't need to make sure we're on cpu (and it defeats the purpose of testing widely).